packages/core/core/src/Graph.js

```diff
@@ -52,6 +56,12 @@ export default class Graph {
     return false;
   }
 
+  getConnectedNodes(node: Node): Array<Node> {
```

Comment:
Should we make a distinction that this only returns upstream nodes?

Reply:
Probably. Have an idea for a name?

Reply:
Upstream and downstream imply a direction relative to the graph rather than relative to the node. Maybe incoming/outgoing would be better? I'm fine either way I guess.

Reply:
Hmm, `incoming`/`outgoing` vs `upstream`/`downstream` seems pretty subjective to me. What if we went super explicit with `getNodesConnectedTo(node)`/`getNodesConnectedFrom(node)`/`replaceNodesConnectedTo(node)`? I know it's typical of me to throw out a long descriptive name, but if the feedback is that we're not quite sure what the method does by the name, then that's what I'm going to suggest.

Reply:
yeah I like that